app/controllers/unpublish_controller.rb

```diff
@@ -4,8 +4,4 @@ class UnpublishController < ApplicationController
   def remove
     @document = Document.with_current_edition.find_by_param(params[:id])
   end
-
-  def withdraw
-    @document = Document.with_current_edition.find_by_param(params[:id])
-  end
```

Comment:
In d0e1620e35b6daa868e8a806e898e79ea032695b I've created a view for users without pre release permissions.  I figured it would be simpler to maintain one withdraw route than two and avoid adding another conditional in the actions partial.